ethstorage/p2p/discovery.go

```diff
@@ -365,6 +365,30 @@ func (n *NodeP2P) DiscoveryProcess(ctx context.Context, log log.Logger, l1ChainI
 	}()
 
 	pstore := n.Host().Peerstore()
+
+	// clear pstore from DB if the version is not match
+	// the DB path is set by p2p.peerstore.path flag, and the default value is esnode_peerstore_db
+	go func() {
```

Comment:
What is the purpose of this function, and in what scenarios would we need to increase the p2pversion?

Reply:
Update function description

![1738738713274](https://github.com/user-attachments/assets/c0e6a730-32c7-4f33-be8c-39bb3944aaec)